src/onboarding/README.md

```diff
@@ -1,2 +1,16 @@
-# onboarding.coderbunker.com
-onboarding process for Coderbunker
+# Onboarding
+
+## Onboarding automation flow
+
+See [this swimlane diagram](https://swimlanes.io/u/DMCO4LO8V)
```

Comment:
you can img src images in the document

Reply:
Do we have an image bed for that? Don't want to upload something in Github repo

Reply:
https://static.swimlanes.io/a6eb6d610445a637450ab126cc97e057.png